app/models/wiki_page.rb

```diff
@@ -11,7 +11,7 @@ class WikiPage < ApplicationRecord
   def default_content
     <<~CONTENT
       ## Lay Summary
-      #{study.brief_summary.description.gsub(/^\s+/, '')}
+      #{study&.brief_summary&.description&.gsub(/^\s+/, '')}
```

Comment:
there was a random spec flake that discovered that this could fail if there isn't a brief summary associated with the description.